eo-runtime/src/test/java/org/eolang/PhiTest.java

```diff
@@ -73,7 +76,8 @@ public void takesStandardPackage() {
     public void takesDirectly() {
         MatcherAssert.assertThat(
             new Dataized(
-                Phi.Φ.attr("org").get().attr("eolang").get().attr("math").get().attr("random").get()
+                Phi.Φ.attr("org").get().attr("eolang").get().attr("math")
+                        .get().attr("random").get().attr("pseudo").get()
```

Comment:
@mximp configure your IDE to always use 4 spaces for all indents. In this module qulice is turned off, that's why this line doesn't break the build.